src/components/ae-overscreen/ae-overscreen.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="ae-overscreen" :class="[{ shadow }, align]">
+    <slot />
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'ae-overscreen',
+  props: {
+    /**
+     * Applies a linear shadow gradient to the overscreen
+     */
+    shadow: Boolean,
+
+    /**
+     * Align the elements either left or right, default alignment is center.
+     * Available values are: `left, right`
+     */
+    align: {
+      type: String,
+      validator: value => [
+        'left',
+        'center',
+        'right',
+      ].includes(value),
+      default: 'center',
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@import '../../styles/globals';
+
+.ae-overscreen {
+  display: flex;
+  align-items: flex-end;
+  justify-content: center;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  padding: rem(20px) rem(32px);
+  z-index: $stack-infinite;
```

Comment:
We should use `z-index` as less as possible, I don't see a reason to use it here.

Reply:
@davidyuk 

I've changed it to `$stack-4` for less priority, I'll think it through a bit later, and see what would work best in this specific component.